Allow for keystore usage (#604)
foundry-rs/starknet-foundry: docs/src/appendix/cast/common.md

Overrides url from `Scarb.toml`.

## `--account, -a <ACCOUNT_NAME>`
Optional.

Account name used to interact with the network, aliased in open zeppelin accounts file.

Overrides account from `Scarb.toml`.

If used with `--keystore`, should be a path to [starkli account JSON file](https://book.starkli.rs/accounts#accounts).

## `--accounts-file, -f <PATH_TO_ACCOUNTS_FILE>`
Optional.

Path to the open zeppelin accounts file holding accounts info. Defaults to `~/.starknet_accounts/starknet_open_zeppelin_accounts.json`.

## `--keystore, -k <PATH_TO_KEYSTORE_FILE>`
Optional.

Path to [keystore file](https://book.starkli.rs/signers#encrypted-keystores).

> If specified, --account should be a path to [starkli account JSON file](https://book.starkli.rs/accounts#accounts).

Reviewer: ```suggestion
When specified, the --account argument must be a path to [starkli account JSON file](https://book.starkli.rs/accounts#accounts).
```

Overrides account from `Scarb.toml`.

> If used with `--keystore`, should be a path to starkli account JSON file.

Reviewer: can we link to some starkli docs, where this file is described or something?

Reviewer: it would be cool if we could do the same with an account file (if possible)
